apps/api/src/services/pubmed-service.ts

```
@@ -0,0 +1,155 @@
+
+		// Step 3: Parse articles
+		const articles: PubMedArticle[] = [];
+		
+		for (const id of ids) {
+			const article = summaryData.result?.[id];
+			if (article) {
+				articles.push({
+					pmid: id,
+					title: article.title || "",
+					abstract: await fetchAbstract(id),
+					authors: article.authors?.map((a: any) => a.name) || [],
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)